network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -45,60 +45,63 @@ <h2 class="h4-heading">{% trans "Featured stories" %}</h2>
           {% card highlight.highlight.image.url highlight.highlight.title highlight.highlight.description highlight.highlight.link_url highlight.highlight.link_label %}
         </div>
       {% endif %} {% endfor %}
-    </div>
-    {% for highlight in page.featured_highlights.all %} {% if forloop.counter > 2 %}
-    <div class="row my-3">
-      <div class="col-sm-12 col-md-4 pt-3 d-none d-md-block">
-        {% homepage_image highlight.highlight.image %}
       </div>
-      <div class="col-sm-12 col-md-8 pt-3">
-        <hr class="mt-0 mb-4">
-        <h4 class="h5-heading mb-3">
-          <a href="{{highlight.highlight.link_url}}">{{highlight.highlight.title}}</a>
-        </h4>
-        <p>{{highlight.highlight.description}}</p>
+      {% for highlight in page.featured_highlights.all %} {% if forloop.counter > 2 %}
+      <div class="row my-3">
+        <div class="col-sm-12 col-md-4 pt-3 d-none d-md-block">
+          {% homepage_image highlight.highlight.image %}
+        </div>
+        <div class="col-sm-12 col-md-8 pt-3">
+          <hr class="mt-0 mb-4">
+          <h4 class="h5-heading mb-3">
+            <a href="{{highlight.highlight.link_url}}">{{highlight.highlight.title}}</a>
```

Comment:
I'm wondering if this is worth using Djangos `{% with %}` template tag to assign this to a better varname, such as:

```
{% with highlight=highlight.highlight %}
    .. {{ highlight.link_url }}
{% endwith %}
```

Could be nicer syntax and _might_ save a few queries when the page loads. 

I'm open to keeping this the way it is. What are you thoughts on this?

Reply:
Yup! Agree using `with` makes the code easier to read. @KalobTaulien do you mind filing a ticket for this so we could separate the work from this PR (for new quote section)?

Reply:
@mmmavis Yep, good call. Ticket opened here: https://github.com/mozilla/foundation.mozilla.org/issues/4914. Marking this as resolved.